env_vars.sh

```diff
@@ -1,4 +1,4 @@
 # Environment variables for build
 OPENBLAS_VERSION="v0.3.7"
 MACOSX_DEPLOYMENT_TARGET=10.9
-CFLAGS="-std=c99 -fno-strict-aliasing"
+CFLAGS="-std=c99 -fno-strict-aliasing -Wl,--strip-debug"
```

Comment:
pass to linker flags only to not break clang build, `LDFLAGS="-Wl,--strip-debug"`

Reply:
done in gh-85